chemist.py

```diff
@@ -34,7 +34,9 @@
     sys.exit(-1)
 
 def verify_signature(body, signature, secret):
-    expected_signature = str(hmac.new(str(secret), msg=body, digestmod=sha1).hexdigest())
+    body = body.encode('utf-8')
+    secret = secret.encode('utf-8')
+    expected_signature = str(hmac.new(secret, msg=body, digestmod=sha1).hexdigest())
```

Comment:
tests pass so :+1:

Reply:
yeah, the new way requires bytes rather than strings